mucommander-core/src/main/java/com/mucommander/ui/action/ActionKeymap.java

```diff
@@ -255,77 +281,77 @@ private static void unregisterAction(MainFrame mainFrame, MuAction action) {
         unregisterActionAccelerators(action, mainFrame.getLeftPanel().getFileTable(), JComponent.WHEN_FOCUSED);
         unregisterActionAccelerators(action, mainFrame.getRightPanel().getFileTable(), JComponent.WHEN_FOCUSED);
     }
-    
+
     /**
      * Unregister MuAction from JComponent.
      */
     private static void unregisterActionAccelerators(MuAction action, JComponent comp, int condition) {
-    	unregisterActionAccelerator(action, action.getAccelerator(), comp, condition);
-    	unregisterActionAccelerator(action, action.getAlternateAccelerator(), comp, condition);
+        unregisterActionAccelerator(action, action.getAccelerator(), comp, condition);
+        unregisterActionAccelerator(action, action.getAlternateAccelerator(), comp, condition);
     }
-    
+
     /**
      * Unregister accelerator of MuAction from JComponent.
      */
-    private static void unregisterActionAccelerator(MuAction action, KeyStroke accelerator, JComponent comp, int condition) {
-    	if(accelerator != null) {
-    		InputMap inputMap = comp.getInputMap(condition);
-    		ActionMap actionMap = comp.getActionMap();
-    		inputMap.remove(accelerator);
-    		actionMap.remove(action.getDescriptor().getId());
-    	}
+    private static void unregisterActionAccelerator(MuAction action,
+            KeyStroke accelerator,
+            JComponent comp,
+            int condition) {
+        if (accelerator != null) {
+            InputMap inputMap = comp.getInputMap(condition);
+            ActionMap actionMap = comp.getActionMap();
+            inputMap.remove(accelerator);
+            actionMap.remove(action.getDescriptor().getId());
```

Comment:
shouldn't we convert it to generic action here as well?

Reply:
another good catch!